bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -54,34 +54,74 @@
+            InsteonBindingConstants.BUTTON_E,
+            InsteonBindingConstants.BUTTON_F,
+            InsteonBindingConstants.BUTTON_G,
+            InsteonBindingConstants.BUTTON_H,
+            InsteonBindingConstants.BROADCAST_ON_OFF,
+            InsteonBindingConstants.CONTACT,
+            InsteonBindingConstants.COOL_SET_POINT,
+            InsteonBindingConstants.DAWNDUSK,
+            InsteonBindingConstants.DIMMER,
+            InsteonBindingConstants.FAN,
+            InsteonBindingConstants.FAN_MODE,
+            InsteonBindingConstants.FAST_ON_OFF,
+            InsteonBindingConstants.FAST_ON_OFF_BUTTON_A,
+            InsteonBindingConstants.FAST_ON_OFF_BUTTON_B,
+            InsteonBindingConstants.FAST_ON_OFF_BUTTON_C,
+            InsteonBindingConstants.FAST_ON_OFF_BUTTON_D,
+            InsteonBindingConstants.HEAT_SET_POINT,
+            InsteonBindingConstants.HUMIDITY,
+            InsteonBindingConstants.HUMIDITY_HIGH,
+            InsteonBindingConstants.HUMIDITY_LOW,
+            InsteonBindingConstants.IS_COOLING,
+            InsteonBindingConstants.IS_HEATING,
+            InsteonBindingConstants.KEYPAD_BUTTON_A,
+            InsteonBindingConstants.KEYPAD_BUTTON_B,
+            InsteonBindingConstants.KEYPAD_BUTTON_C,
+            InsteonBindingConstants.KEYPAD_BUTTON_D,
+            InsteonBindingConstants.KEYPAD_BUTTON_E,
+            InsteonBindingConstants.KEYPAD_BUTTON_F,
+            InsteonBindingConstants.KEYPAD_BUTTON_G,
+            InsteonBindingConstants.KEYPAD_BUTTON_H,
+            InsteonBindingConstants.KWH,
+            InsteonBindingConstants.LAST_HEARD_FROM,
+            InsteonBindingConstants.LED_BRIGHTNESS,
+            InsteonBindingConstants.LED_ONOFF,
+            InsteonBindingConstants.LIGHT_DIMMER,
+            InsteonBindingConstants.LIGHT_LEVEL,
+            InsteonBindingConstants.LOAD_DIMMER,
+            InsteonBindingConstants.LOAD_SWITCH,
+            InsteonBindingConstants.LOAD_SWITCH_FAST_ON_OFF,
+            InsteonBindingConstants.LOAD_SWITCH_MANUAL_CHANGE,
+            InsteonBindingConstants.LOWBAT,
+            InsteonBindingConstants.MANUAL_CHANGE,
+            InsteonBindingConstants.MANUAL_CHANGE_BUTTON_A,
+            InsteonBindingConstants.MANUAL_CHANGE_BUTTON_B,
+            InsteonBindingConstants.MANUAL_CHANGE_BUTTON_C,
+            InsteonBindingConstants.MANUAL_CHANGE_BUTTON_D,
+            InsteonBindingConstants.NOTIFICATION,
+            InsteonBindingConstants.ON_LEVEL,
+            InsteonBindingConstants.RAMP_DIMMER,
+            InsteonBindingConstants.RAMP_RATE,
+            InsteonBindingConstants.RESET,
+            InsteonBindingConstants.STAGE1_DURATION,
+            InsteonBindingConstants.SWITCH,
+            InsteonBindingConstants.SYSTEM_MODE,
+            InsteonBindingConstants.TEMPERATURE,
+            InsteonBindingConstants.TOP_OUTLET,
+            InsteonBindingConstants.UPDATE,
+            InsteonBindingConstants.WATTS)
```

Comment:
I assume this was accidental? Eclipse formatting produces the original formatting.

Reply:
do you mean with multiple on each line?  I had a really hard time reading this so I changed to one per line similar to formatting in InsteonDeviceHandler.java.  I can set to however you want.

Reply:
This is not following how it is formatted when using the code formatters built into eclipse

Reply:
done